<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>160</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>670</width>
    <height>160</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>670</width>
    <height>160</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Печать талонов на отправку</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="name_group">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>401</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Название</string>
    </property>
    <widget class="QLineEdit" name="name_edit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>381</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="serial_group">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>401</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Серийный номер</string>
    </property>
    <widget class="QLineEdit" name="serial_edit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>381</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="warranty_group">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Гарантия</string>
    </property>
    <widget class="QComboBox" name="warranty_combo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="price_group">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>100</y>
      <width>120</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Цена</string>
    </property>
    <widget class="QLineEdit" name="price_edit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>грн.</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="add_button">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>110</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="print_button">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>110</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QListWidget" name="elements_view">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>256</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
